### ISSUE TITLE
Can't seem to find the Cloudformation template when I click link provided

### ISSUE THREAD

**Issue by Joel-fogue** [awslabs/ecs-demo-php-simple-app#12]
If you go to step 3 of this link: https://docs.aws.amazon.com/AWSGettingStartedContinuousDeliveryPipeline/latest/GettingStarted/ECS_CD_Pipeline.html

and you click that Launch Stack, the template is nowhere to be found; was it moved somewhere else maybe?

**Comment by diegom626** [comment]
Yes, i think they moved it. here is the repo and all the templates.
https://github.com/awslabs/ecs-refarch-continuous-deployment
